filebeat/filebeat.full.yml

```diff
@@ -686,7 +686,7 @@ output.elasticsearch:
 #logging.level: error
 
 # Enable debug output for selected components. To enable all selectors use ["*"]
-# Other available selectors are beat, publish, service
+# Other available selectors are "beat", "publish", "service"
 # Multiple selectors can be chained.
 #logging.selectors: [ ]
```

Comment:
No, you are right. In the full we have always the defaults. 

Perhaps it would be make sense to change our default here, because if you enable debug I would assume that it is somehow expected to output messages. But currently default = "no selectors" as far as I understand?